Reply/app/src/main/res/color/star_statelist.xml

```diff
@@ -12,12 +12,12 @@
   or implied. See the License for the specific language governing permissions and limitations under
   the License.
   -->
-<resources>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
 
-    <dimen name="plane_00">0dp</dimen>
-    <dimen name="plane_01">1dp</dimen>
-    <dimen name="plane_02">2dp</dimen>
-    <dimen name="plane_06">6dp</dimen>
-    <dimen name="plane_08">8dp</dimen>
-    <dimen name="plane_16">16dp</dimen>
-</resources>
\ No newline at end of file
+    <item
+        android:state_checked="true"
+        android:color="#F2D053" />
```

Comment:
Let's talk to design and see what this color should be in light v. dark v. dynamic and whether we should use color harmonization. Ideally we don't use a hard coded hex value

Reply:
Alright for sure. This may block this PR, so for now I'll create a bug and I'll create a TODO.

Reply:
Just messaged design.